typescript/libs/popup_manager.py

```diff
@@ -263,13 +263,43 @@ def encode(str, kind):
         else:
             activeParam = ''
 
+        theme_styles = self.get_theme_styles()
+
         return {"signature": signature,
                 "description": description,
                 "activeParam": activeParam,
                 "index": "{0}/{1}".format(self.signature_index + 1,
                                           len(self.signature_help["items"])),
                 "link": "link",
-                "fontSize": PopupManager.font_size}
+                "fontSize": PopupManager.font_size,
+                "typeStyles": theme_styles["type"],
+                "keywordStyles": theme_styles["keyword"],
+                "nameStyles": theme_styles["name"],
+                "paramStyles": theme_styles["param"],
+                "textStyles": theme_styles["text"]}
+
+    def format_css(self, style):
```

Comment:
Can you pull these identical helpers into one place?

Reply:
Sure, I'll look at that when I get home later

Reply:
Okay, I pulled the functions for figuring out the styles out into their own file